Lib/typing.py

```diff
@@ -486,7 +486,7 @@ def open_helper(file: str, mode: MODE) -> str:
     """)
 
 
-class ForwardRef(_Final, _root=True):
+class ForwardRef(_Final, _Immutable, _root=True):
```

Comment:
Why is this needed? This will preserve _identity_ (not just value) after copying.

Reply:
Well, it's not strictly needed, but there's no reason that two references to the same thing should refer to different objects.

Reply:
> Well, it's not strictly needed

Then don't do this. This can break things in unexpected ways. Also this "thing" has internal state that does change.